components/message/__test__/index.test.js

```diff
@@ -0,0 +1,102 @@
+import message from '..'
+import MessageBox from '../message-box'
+import Message from '../message'
+import { creatComponentVm, creatVueVm, simulateEvent, nextTick, renderVmString } from 'util.js'
+const delay = timeout => new Promise(resolve => setTimeout(resolve, timeout))
+
+describe('message', () => {
+  afterEach(() => {
+    message.destroy()
+    message.config({
+      prefixCls: 'ant-message',
+      duration: 3
+    })
+  })
+
+  it('message-box should has prefixCls class', () => {
+    const vm = creatComponentVm(MessageBox, {
+      prefixCls: 'test-message-box'
+    })
+    expect(vm.$el.classList.contains('test-message-box')).toBeTruthy()
+  })
+
+  it('should has prefixCls class', () => {
+    const vm = creatComponentVm(Message, {
+      prefixCls: 'test-message'
+    })
+    expect(vm.$el.classList.contains('test-message-notice')).toBeTruthy()
+  })
+
+  it('should be able to args onClose', async () => {
+    let closeFlag = 1
+    const userOnClose = () => closeFlag = 2
```

Comment:
ESLint: Arrow function should not return assignment. (no-return-assign).
https://github.com/eslint/eslint/issues/5150